priv/repo/seeds.exs

```diff
@@ -689,6 +689,17 @@ giant_fruit_params = %{
 {:ok, giant_fruit} =
   GameBackend.Items.create_consumable_item(giant_fruit_params)
 
+polymorph_params = %{
+  active: true,
```

Comment:
We have to set this to false, otherwise the game will crash since the client needs an image to display the items in the UI 

<img width="597" alt="Screenshot 2024-08-05 at 17 59 21" src="https://github.com/user-attachments/assets/594a76d3-a970-41ca-82a2-536efea1b4ec">

Reply:
Whoops, done!